File: app/src/main/java/me/devsaki/hentoid/fragments/about/LicensesFragment.java
package me.devsaki.hentoid.fragments.about;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.webkit.WebView;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.appcompat.widget.Toolbar;
import androidx.fragment.app.Fragment;

import me.devsaki.hentoid.R;

import static androidx.core.view.ViewCompat.requireViewById;

public class LicensesFragment extends Fragment {

    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View rootView = inflater.inflate(R.layout.fragment_licenses, container, false);

        Toolbar toolbar = requireViewById(rootView, R.id.licenses_toolbar);
        toolbar.setNavigationOnClickListener(v -> requireActivity().onBackPressed());

        WebView webView = requireViewById(rootView, R.id.licenses_web_view);
        webView.loadUrl("file:///android_asset/licenses.html");

        return rootView;
    }
}

File: app/src/main/java/me/devsaki/hentoid/fragments/about/LicensesFragment.kt
package me.devsaki.hentoid.fragments.about

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import me.devsaki.hentoid.R
import me.devsaki.hentoid.databinding.FragmentLicensesBinding

class LicensesFragment : Fragment(R.layout.fragment_licenses) {

    private var _binding: FragmentLicensesBinding? = null
    private val binding get() = _binding!!

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        _binding = FragmentLicensesBinding.inflate(inflater, container, false)
        return binding.root
    }

    override fun onDestroyView() {
        super.onDestroyView()
        _binding = null
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        binding.licensesToolbar.setNavigationOnClickListener { requireActivity().onBackPressed() }
        binding.licensesWebView.loadUrl("file:///android_asset/licenses.html")
    }
}